eclair-front/src/main/resources/application.conf

```diff
@@ -0,0 +1,86 @@
+    // front is really using the expected key
+    pub = ${NODE_PUB_KEY}
+  }
+}
+
+akka {
+  actor.provider = cluster
+  remote.artery {
+    canonical.hostname = "127.0.0.1"
+    canonical.hostname = ${?LOCAL_IP} // this will override the default value with the env variable if set
+    canonical.port = 25520
+
+    untrusted-mode = on
+    trusted-selection-paths = [
+      "/system/cluster/core/daemon",
+      "/system/cluster/heartbeatReceiver",
+      "/system/distributedPubSubMediator",
+      "/system/clusterReceptionist/replicator"
+    ]
+
+    advanced {
+      outbound-message-queue-size = 30720 // 10x default because of sync
+    }
+  }
+  cluster {
+    shutdown-after-unsuccessful-join-seed-nodes = 10s # front won't start if back is offline
+    roles = [frontend]
+    seed-nodes = ["akka://eclair-node@"${BACKEND_IP}":25520"]
+  }
+  coordinated-shutdown.terminate-actor-system = on
+  coordinated-shutdown.exit-jvm = on
+  //It is recommended to load the extension when the actor system is started by defining it in akka.extensions
+  //configuration property. Otherwise it will be activated when first used and then it takes a while for it to be populated.
+  extensions = ["akka.cluster.pubsub.DistributedPubSub"]
+
+}
+
+kamon {
+  environment.host = ${HOSTNAME}
+  instrumentation.akka {
+    filters {
+      actors {
+        # Decides which actors generate Spans for the messages they process, given that there is already an ongoing trace
+        # in the Context of the processed message (i.e. there is a Sampled Span in the Context).
+        #
+        trace {
+          includes = []
+          excludes = ["**"] # we don't want automatically generated spans because they conflict with the ones we define
+        }
+      }
+    }
+  }
+}
+
+akka {
+
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  logger-startup-timeout = 30s
+  loglevel = "DEBUG" # akka doc: You can enable DEBUG level for akka.loglevel and control the actual level in the SLF4J backend without any significant overhead, also for production.
+  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
+
+  io {
+    tcp {
+      # The maximum number of bytes delivered by a `Received` message. Before
+      # more data is read from the network the connection actor will try to
+      # do other work.
+      # The purpose of this setting is to impose a smaller limit than the
+      # configured receive buffer size. When using value 'unlimited' it will
+      # try to read all from the receive buffer.
+      # As per BOLT#8 lightning messages are at most 2 + 16 + 65535 + 16 = 65569bytes
+      # Currently the largest message is update_add_htlc (~1500b).
+      # As a tradeoff to reduce the RAM consumption, in conjunction with tcp pull mode,
+      # the default value is chosen to allow for a decent number of messages to be prefetched.
+      max-received-message-size = 16384b
```

Comment:
Actually this isn't accurate anymore, gossip messages can fill up to the 64k limit, can't they?

Reply:
Here is the doc for that configuration parameter:
```
      # The maximum number of bytes delivered by a `Received` message. Before
      # more data is read from the network the connection actor will try to
      # do other work.
      # The purpose of this setting is to impose a smaller limit than the
      # configured receive buffer size. When using value 'unlimited' it will
      # try to read all from the receive buffer.
      max-received-message-size = unlimited
```

I think we should use a multiple of the max message size, maybe 262144 bytes, WDYT?

Reply:
I agree, that's a good starting parameter :+1: 
And we'll adjust if we see issues related to that.

Reply:
Done in 06d4d12.